packages/runtime/src/useCases/consumption/attributes/AttributeSuccessionResponse.ts

```diff
@@ -1,6 +1,13 @@
+import { CoreId } from "@nmshd/transport";
 import { LocalAttributeDTO } from "../../../types";
 
 export interface AttributeSuccessionResponse {
     predecessor: LocalAttributeDTO;
     successor: LocalAttributeDTO;
 }
+
+export interface AttributeSuccessionWithNotificationResponse {
+    predecessor: LocalAttributeDTO;
+    successor: LocalAttributeDTO;
+    notificationId: CoreId;
```

Comment:
we never return CoreIds from DTOs or responses. In this case we need a string

Reply:
Only we currently do in NotifyPeerAboutRepositoryAttributeSuccessionResponse and SucceedRelationshipAttributeAndNotifyPeerResponse. Independently of the result of the discussion below, should I change the type of notificationId?

Reply:
yes, please!